src/main/java/website/frontrow/Launcher.java

```diff
@@ -181,6 +174,72 @@ private void startScheduler(Game game)
                 }
             });
         }
+        else
+        {
+        	map.put(KeyEvent.VK_A, () ->
+            {
+                addToLog("[KEY]\t< \'<-\' > Pressed.");
+                game.getPlayers().get(0).goLeft();
+            });
+
+            map.put(KeyEvent.VK_D, () ->
+            {
+                addToLog("[KEY]\t< \'->\' > Pressed.");
+                game.getPlayers().get(0).goRight();
+            });
+
+            map.put(KeyEvent.VK_W, () ->
+            {
+                addToLog("[KEY]\t< \' \' > Pressed.");
+                game.getPlayers().get(0).jump();
+            });
+
+            map.put(KeyEvent.VK_SPACE, () ->
+            {
+                addToLog("[KEY]\t< \'Z\' > Pressed.");
+
+                if(game.isRunning())
+                {
+                    Player p = game.getPlayers().get(0);
+                    game.getLevel().addUnit(
+                            new Bubble(p.getLocation(),
+                                    new Point(p.getDirection().getDeltaX() * 4, 0),
```

Comment:
These are the controls for firing a bubble by player 1.
I do see a small issue though, so I'll fix that.